src/data/graphql/queries.graphql

```diff
@@ -466,6 +466,33 @@ query ColonyMembersWithReputation($colonyAddress: String!, $domainId: Int) {
   colonyMembersWithReputation(colonyAddress: $colonyAddress, domainId: $domainId) @client
 }
 
+query ContributorsAndWatchers($colonyAddress: String!, $colonyName: String!, $domainId: Int) {
+  contributorsAndWatchers(colonyAddress: $colonyAddress, colonyName: $colonyName, domainId: $domainId) @client {
+    contributors {
+      id
+      directRoles
+      roles
+      banned
```

Comment:
So now that we've split them, I don't think "banning" a contributor makes sense anymore, especially since this is being enforced only in the UI and only in comments, while at the contract level they still have free reign.

This is a thing we need to bring up with product and discuss the spec change, while keeping in mind _Verified Recipients_ _(Address Book)_ and how that all fits into this

Reply:
Good point. I still see value in being able to ban both contributors and watchers from commenting.

I will bring it up during a Product Sync for discussion though.